Documentation/ApiOverview/Database/DatabaseStructure/Index.rst

```diff
@@ -1,124 +1,141 @@
+    not be used (for example, disabled backend users, content not visible in the
+    frontend).
 
-   Except for the "uid" and "pid" fields, all other fields do not fill a role
-   automatically as soon as they exist. Their existence must be declared in
-   the :ref:`TCA configuration <t3tca:ctrl>`. This means that such fields can
-   also be named freely, the above are the default names TYPO3 uses - for
-   consistency it is recommended to name them that way.
+..  note::
+    With the exception of the :sql:`uid` and :sql:`pid` fields, all other fields
+    do not automatically fill a role as soon as they exist. Their existence must
+    be declared in the :ref:`TCA configuration <t3tca:ctrl>`. This means that
+    such fields can also be named freely, the above are the default names TYPO3
+    uses - for consistency reasons it is recommended to name them that way.
 
 
-.. index:: Tables; pages
-.. _database-structure-pages:
+..  index:: Tables; pages
+..  _database-structure-pages:
 
 The "pages" table
 =================
 
-The `pages` table has a special status: It is the backbone of TYPO3 CMS, as it provides
-the hierarchical page structure into which all other TYPO3 CMS managed records are positioned.
-All other managed tables in TYPO3 have a `pid` field that points to a `uid` record in this table.
-So any managed table record in TYPO3 is always positioned on exactly one page in the page tree.
-This makes the `pages` table the mother of all other managed tables. It can be seen as a directory
-tree with all other table records as files.
-
-Standard pages are quite literally web site pages in the frontend. But they can also be storage
-spaces in the backend, very much like folders on a hard disk. For any record, the `pid` field
-contains a reference to the page where that record is stored. For pages, the `pid` fields behaves
-as a reference to their parent pages.
-
-The special "root" page has some unique properties: its pid is 0 (zero), it does not exist as
-a row in the `pages` table, only admin-users can access records on it and these records have
-to be :ref:`explicitly configured to reside in the root page <t3tca:ctrl-reference-rootlevel>` - usually
-table records may only be created on a real page.
-
-.. index::
-   Tables; MM relations
-   Tables; Cache
-   Tables; System information
-.. _database-structure-other-tables:
+The :sql:`pages` table has a special status: It is the backbone of TYPO3, as it
+provides the hierarchical page structure into which all other records managed by
+TYPO3 are placed. All other managed tables in TYPO3 have a :sql:`pid`
+field that points to a :sql:`uid` record in this table. Thus, each managed table
+record in TYPO3 is always placed on exactly one page in the page tree. This
+makes the :sql:`pages` table the mother of all other managed tables. It can be
+seen as a directory tree with all other table records as files.
+
+Standard pages are literally website pages in the frontend. But they can also be
```

Comment:
What does "they" refer to? the standard pages?

Reply:
Yup, I understand it so.

Reply:
But wouldn't it usually be non-standard pages, sys folders and such that contain the records? If not we should maybe emphasize that your meaning here is that standard pages can also be used for that purpose (then we should maybe also give a use case?)

Reply:
No, for example, I have often records on standard pages (think of IRRE elements). Also pages are records. When you open that you have to use something like: "Standard pages, backend user pages, and - depending on your configuration - custom doktypes ...". Here it is about database relations, not the do's and dont's of how to store records into folders or pages.